t/02_test_tests.t

```diff
@@ -0,0 +1,85 @@
+use strict;
+use warnings;
+use Test::Builder::Tester tests => 10;
+use Test::Most;
+use Email::Stuffer;
+use Email::Stuffer::TestLinks;
+use Email::Sender::Transport::Test;
```

Comment:
I'd be tempted to list [Email::Sender::Transport::Test](https://metacpan.org/pod/Email::Sender::Transport::Test) in `cpanfile` explicitly, even though it's pulled in via Email::Stuffer's dep chain.

Reply:
Done, it's only used by tests however